### DISCUSSION TITLE
Regarding ksponspeech

### DISCUSSION THREAD

**Discussion by qkrguswn2401** [speechbrain/speechbrain#1655] [Q&A]
hi, I am having trouble preparing data for ksponspeech.   (https://github.com/speechbrain/speechbrain/tree/89553c919393ebf45046a47493a15be359cbd009/recipes/KsponSpeech)
 I can't run ksponspeech_prepare.py correctly. 
I have created a main function to prepare data as below.
![image](https://user-images.githubusercontent.com/36841216/150749171-5a819fde-2396-49de-a189-5c4c58d8fe8b.png)
The text_dict is empty.  What am I doing wrong?
![image](https://user-images.githubusercontent.com/36841216/150749353-6d2bd03d-f824-4acc-85ec-6ebb116c101c.png)

can you share *.trn if you could?

**Comment by ddwkim** [comment]
Hi,

.trn files are needed as they include transcriptions of each wav file. 
These files are provided with the corpus.
If you have .txt files as transcriptions of each wav file, then another way around may be changing the ksponspeech_prepare.py code in a way that parses .txt files instead of .trn files.
All you need to train the models are train.csv, eval_clean.csv, eval_other.csv, dev.csv files, which contains [ID, start, end, duration, text, wav path] of each wav file as rows.
Hope this helps.
